cmd/agent/subcommands/run/settings.go

```diff
@@ -34,5 +34,5 @@ func initRuntimeSettings() error {
 	if err := commonsettings.RegisterRuntimeSetting(commonsettings.ProfilingGoroutines("internal_profiling_goroutines")); err != nil {
 		return err
 	}
-	return commonsettings.RegisterRuntimeSetting(commonsettings.ProfilingRuntimeSetting("internal_profiling"))
+	return commonsettings.RegisterRuntimeSetting(commonsettings.ProfilingRuntimeSetting{SettingName: "internal_profiling", Service: "datadog-agent"})
```

Comment:
Previously the service name would be set to `datadog-agent` [here](https://github.com/DataDog/datadog-agent/blob/41a54d0772ee937f533f47207a10b13b27a7658d/pkg/config/settings/runtime_setting_profiling.go#L75), so this preserves existing behavior